### PR TITLE
Ej/background maps settings flag

### PR DESCRIPTION
**Pull request by lightlii**
Adds 'Experiments' list to settings menu with `Background Maps` as (currently only) option.
When Background Maps flag is activated, Background maps becomes an available option in settings.

closes https://github.com/digidem/mapeo-desktop/issues/733

## Preview

Screenshot from 2023-07-17 12-33-16
Screenshot from 2023-07-17 12-33-10
Screenshot from 2023-07-17 12-33-25

### CODE REVIEW

**Review by ErikSin — CHANGES_REQUESTED** [review]
Good job! I left some minor comments/suggestions

2 things:
1. Once you rebase/merge from master, I would like to give it one more quick look before you merge.
2. I added alot of maps in testing. When it is loading the list of maps, it blocks the UI. I think that is fine when it is initially loading, but Tan stack query makes the API call everytime the window is out of focus and refocused, causing the UI to be blocked. I think we should [disable invalidating the query](https://tanstack.com/query/v4/docs/react/guides/disabling-queries). And when the users uploads a new map, we can use the `refetch` function to refresh the list. That way the list is not constantly trying to refetch even when we know there is no new data.

File: src/renderer/components/dialogs/ImportMapStyle.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        <Typography variant='body1' className={classes.title}>
+          {t(m.addMap)}
+        </Typography>
+        <IconButton onClick={close}>
+          <CloseIcon />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent>
+        <Card>
+          <CardActionArea style={{ backgroundColor: '#EDEDED', padding: 30 }} onClick={selectMbTileFile}>
```
[the remark this answers is not in the document]
Comment (lightlii) [reply src/renderer/components/dialogs/ImportMapStyle.js]: Yep, I'd copied over the code from the prev PR, it was using electron `remote` which no longer works.

Fixed now!

**Comment by lightlii** [comment]
Hey I think I've added what you requested in (2) in your comment above. So it's ready to look at again I'd say!

**Comment by lightlii** [comment]
Oops sorry, nope I missed error handling in `ImportMapStyle`, I can finish that up first thing tomorrow

**Comment by lightlii** [comment]
@ErikSin I've rebased from master, so this is ready for a last look over

**Comment by ErikSin** [comment]
Im unable to open it. Are you able to open it on your local device?

**Comment by lightlii** [comment]
> Im unable to open it. Are you able to open it on your local device?

I don't have any issues running it. Lets look at that together when we call later

**Comment by ErikSin** [comment]
There are some import errors that the linter and vs code are catching

**Comment by lightlii** [comment]
These changes are in #744 so @ErikSin & I agreed to close and only review #744

**Pull request closed without merging**